src/input_processor.py

```diff
@@ -0,0 +1,572 @@
+import subprocess
```

Comment:
is this Workbench library? Or did we copy it here for convenience? (it this allowed?)

Reply:
Moved inside the function now